hadoop-hdds/docs/content/design/how-to-do-a-nonrolling-upgrade.md

```diff
@@ -0,0 +1,49 @@
+---
+title: How to do a non-rolling upgrade of Ozone 
```

Comment:
These steps should also be a page on the wiki for performing non-rolling upgrades.

Reply:
Yes, will work with @errose28 to add wiki doc as well.